lib/subprocess.js

```diff
@@ -9,6 +9,16 @@ import { quote } from 'shell-quote';
 import _ from 'lodash';
 
 
+const MAX_LINE_PORTION_LENGTH = 0xFFFF;
```

Comment:
would be good to have a comment explaining why this exists (so that we don't use too much memory), and also why `0xFFFF` is a good choice of constant value.

Reply:
added a comment

64KB is just a good number :)